.github/workflows/publish.yml

```
@@ -0,0 +1,40 @@
+  cancel-in-progress: false
+
+# Default to bash
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    uses: ./.github/workflows/_hugo.yml
+    with:
+      HUGO_VERSION: '0.134.2'
```

actionlint

input "HUGO_VERSION" is not defined in "./.github/workflows/_hugo.yml" reusable workflow. no input is defined

(workflow-call)